docs/customization/custom-scripts.md

```diff
@@ -142,6 +142,19 @@ obj.full_clean()
 obj.save()
 ```
 
+## Error handling
+
+Sometimes things go wrong and a script will run into an `Exception`. If that happens and an uncatched Exception is raised by the custom script, the execution is aborted and a full stacktrace reported.
+
+Although this is helpful for debugging in some situations it might be required to cleanly abort the execution of a custom script (e.g. because of invalid input data) and thereby make sure no changes are performed on the database. In this case the script can throw a `AbortScript` exception, which will prevent the stacktrace from being reported, but still terminating the script's execution and reporting a given error message.
```

Comment:
```suggestion
Although this is helpful for debugging, in some situations it might be required to cleanly abort the execution of a custom script (e.g. because of invalid input data) and thereby make sure no changes are performed on the database. In this case the script can throw an `AbortScript` exception, which will prevent the stack trace from being reported, but still terminating the script's execution and reporting a given error message.
```